com.github.Inochi2D.Inochi-Creator.yml

```diff
@@ -0,0 +1,167 @@
+id: com.github.Inochi2D.Inochi-Creator
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ldc
+  - org.freedesktop.Sdk.Extension.dmd
+command: "inochi-creator"
+finish-args:
+    - "--device=all" #OpenGL rendering, webcams (may be required? not sure)
+    - "--socket=x11"
+    - "--share=ipc" # The X and Wayland sockets are permanently exposed, because kdialog takes longer to start if it only has X access, and doesn't work at all if only using Wayland.
+    - "--socket=wayland"
+    - "--filesystem=host" #This is a dev tool, it should probably have as much access to the host as it can. It also doesn't yet utilize portals.
+    - "--env=XDG_CURRENT_DESKTOP=KDE" #Hopefully fixes the issues it has with running on non-KDE desktops.

```

Comment:
what issues? You shouldn't be setting this.

Reply:
Addressed in the manifest, also see https://github.com/Inochi2D/inochi-creator/issues/76